src/project/qml/MuseScore/Project/internal/Export/ExportOptionItem.qml

```diff
@@ -24,14 +24,14 @@ import QtQuick.Layouts 1.15
 
 import MuseScore.UiComponents 1.0
 
-RowLayout {
+ColumnLayout {
     id: root
 
     property int labelWidth: 72
     property alias text: label.text
 
     width: Math.min(implicitWidth, parent.width)
-    spacing: 12
+    spacing: 8
 
     StyledTextLabel {
         id: label
```

Comment:
On the line below this one\*, I think `Layout.preferredwidth: root.labelWidth` should be replaced with `Layout.fillWidth: true`, since now the label may fill the whole width of the dialog if necessary. After that, it seems like `property int labelWidth` can be removed altogether. 

\* I'm posting the comment here because GitHub doesn't let me comment on the line I mean because that line is too far away from the modified lines...

Reply:
Good point, thank you!